modules/scripts-utils/dtbook-utils/src/main/resources/xml/fix-dtbook/fix-dtbook.xpl

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<p:declare-step xmlns:p="http://www.w3.org/ns/xproc" version="1.0"
+                xmlns:px="http://www.daisy.org/ns/pipeline/xproc"
+                xmlns:cx="http://xmlcalabash.com/ns/extensions"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                type="px:dtbook-fix" name="main">
+    <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+        <h1 px:role="name">Fix DTBook XML</h1>
+        <p px:role="desc">Apply fixing routines on a given dtbook xml.</p>
+    </p:documentation>
+
+    <!-- Import -->
+    <p:import href="repair.xpl"/>
+    <p:import href="tidy.xpl"/>
+    <p:import href="narrator.xpl"/>
+
+    <!-- I/O -->
+    <p:input port="source" px:media-type="application/x-dtbook+xml">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <p>A single DTBook document</p>
+        </p:documentation>
+    </p:input>
+    <p:output port="result" px:media-type="application/x-dtbook+xml">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <p>The result DTBook document</p>
+        </p:documentation>
+    </p:output>
+
+    <!-- Options -->
+    <p:option name="repair" required="false" select="false()">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Repair the dtbook</h2>
+            <p px:role="desc">Apply repair routines on the dtbook</p>
+        </p:documentation>
+    </p:option>
+    <p:option name="fixCharset" required="false" select="false()">
+        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+            <h2 px:role="name">Repair - Fix Charset</h2>
+            <p px:role="desc">Fix the document charset (To be implemented)</p>
```

Comment:
Is this option not implemented yet? Let's drop it from the script options then (comment it out).

Reply:
@NPavie Is this option going to be implemented or should we drop it?